verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -212,6 +232,19 @@ void VerilogPreprocess::RegisterMacroDefinition(
   // TODO(hzeller): multiline warning with 'previously defined here' location
 }
 
+absl::Status VerilogPreprocess::ExpandMacro(MacroDefinition& definition){
```

Comment:
Don't pass parameters to be modified by reference, but rather as pointer. This is the style used throughout so that it is easy to see at the call-site for the reader that something is modified.